packages/flutter/lib/src/material/stepper.dart

```diff
@@ -671,11 +671,14 @@ class _StepperState extends State<Stepper> with TickerProviderStateMixin {
     assert(debugCheckHasMaterialLocalizations(context));
     assert(() {
       if (context.ancestorWidgetOfExactType(Stepper) != null)
-        throw FlutterError(
-          'Steppers must not be nested. The material specification advises '
-          'that one should avoid embedding steppers within steppers. '
-          'https://material.io/archive/guidelines/components/steppers.html#steppers-usage'
-        );
+        throw FlutterError.fromParts(<DiagnosticsNode>[
```

Comment:
Go ahead and revert this file. The previous error is equivalent except for likely missing  a \n after "Nested."

Reply:
Reverting this file but should I keep the \n after "nested." word in the first line?
If not then I will have to revert the change in the test file as well